src/widgets/algorithm_widget.py

```diff
@@ -0,0 +1,61 @@
+from PyQt5 import QtCore, QtWidgets
+from widgets.unfold_widget import UnfoldWidget
+
+
+class AlgorithmWidget(UnfoldWidget):
+    def __init__(self, parent):
+        super().__init__(parent)
+
+        # unfold button
+        self.button.setText("ALGORITHM")
+        self.button.setStyleSheet("background-color: rgb(177, 221, 240);")
+        self.button.clicked.connect(lambda: self.parent().unfold(2))
+
+        # algorithm frame
+        self.frame.setStyleSheet("background-color: #EBF4F7;")
+
+        # exploration technique selection
+        self.technique_group = QtWidgets.QGroupBox(self.frame)
+        self.technique_group.setGeometry(QtCore.QRect(30, 90, 221, 111))
+        self.technique_group.setTitle("Exploration technique")
+
+        self.technique_box = QtWidgets.QComboBox(self.technique_group)
+        self.technique_box.setGeometry(QtCore.QRect(30, 50, 121, 41))
+        self.technique_box.setStyleSheet("color: rgb(0,0,0);")
+        self.technique_box.addItem("clustering")
+        self.technique_box.addItem("associations")
```

Comment:
Maybe some helper component that will handle selection with list of choices passed?